rust/agama-locale-data/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+use serde::Deserialize;
+use std::fs::File;
+use std::io::BufReader;
+use quick_xml::de::Deserializer;
+use flate2::bufread::GzDecoder;
+
+pub mod keyboard;
+pub mod language;
+pub mod territory;
+pub mod timezone_part;
+pub mod ranked;
+
+pub fn get_keyboards() -> keyboard::Keyboards {
+    const FILE_PATH: &str = "/usr/share/langtable/data/keyboards.xml.gz";
```

Comment:
I would put all the constants outside of the functions so it is easier to adjust the names. Although I would expect Rust not define the const on every single call.